ci-scripts/yocto.groovy

```diff
@@ -216,8 +236,15 @@ void buildManifest(String variantName, String imageName, boolean smokeTests=fals
             boolean buildUpdate = variantName.startsWith("rpi")
             buildImageAndSDK(yoctoDir, imageName, buildUpdate)
             if (smokeTests) {
-                runSmokeTests(yoctoDir, imageName, testBitbake)
+                boolean weekly = env.WEEKLY_BUILD == "true"
+		runSmokeTests(yoctoDir, imageName)
+		if(weekly) {
+		    run runBitbakeTests(yoctoDir)
+		}				
+
```

Comment:
Yes, this is what I was thinking of as a solution.
The indentation and whitespace looks a bit strange though.

Reply:
ok. Will fix it once im back. I was working on vim so didnt notice it.